buildkite/src/Command/Promotion/VerifyDocker.dhall

```diff
@@ -36,8 +36,8 @@ let promoteDockerVerificationStep =
                                 spec.remove_profile_from_name
 
                         in  Cmd.run
-                              "docker pull ${repo}/${Artifact.dockerName
-                                                       spec.name}:${new_tag}"
+                              ". ./buildkite/scripts/export-git-env-vars.sh && NEW_TAG=\$(echo \"${new_tag}\" | sed 's/[^a-zA-Z0-9_.-]/-/g') && docker pull ${repo}/${Artifact.dockerName
```

Comment:
Similar to promoting, here we want to evaluate env vars which are possibly the part of new_tag string. I also added some conversion of illegal chars